wordpress-mobile/WordPress-Android
# Moves VersionUtils to WordPress-Utils-Android

### Depends on https://github.com/wordpress-mobile/WordPress-Utils-Android/pull/90

To test:
Verify that the app does not crash and the [tests](https://github.com/wordpress-mobile/WordPress-Utils-Android/pull/90/files#diff-f3effa178b391d32530469b7869104f8018c4eaf06c1e24beabfad6d59ff578a) are passing

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Used the existing (moved) automated [tests](https://github.com/wordpress-mobile/WordPress-Utils-Android/pull/90/files#diff-f3effa178b391d32530469b7869104f8018c4eaf06c1e24beabfad6d59ff578a)

3. What automated tests I added (or what prevented me from doing so)
Relied on existing tests

PR submission checklist:
